src/posts/keymaps/haraldkubota.md

```diff
@@ -0,0 +1,24 @@
+---
+author: haraldkubota
+baseLayouts: [QWERTY]
+firmwares: [QMK]
+hasHomeRowMods: true
+hasLetterOnThumb: false
+hasRotaryEncoder: false
+isAutoShiftEnabled: false
+isComboEnabled: true
+isSplit: true
+isTapDanceEnabled: true
+keybindings: [Vim]
+keyboard: Corne
+keyCount: 46
+keymapImage: https://raw.githubusercontent.com/haraldkubota/static-files/master/corne/corne-v4.png
+keymapUrl: https://github.com/haraldkubota/static-files/blob/5f23e24eaa18bf1ff087ef41f7909e58bfc91d4f/corne/
+languages: [English]
+layerCount: 4
+OS: [Linux, Windows]
+stagger: columnar
+summary: A 42 key ergonomic columnar layout for corne-style split keyboards. After weeks of changing the layout, this worked best for me.
```

Comment:
Wouldn't it be more appropriate to set `keyCount: 43` and say `A 43 key ergonomic ...` in the description considering that you're using all the usual 42 keys of a corne + 1 extra key for the TG(5) ?

Reply:
Interesting and valid point. I do only use 43 keys, but mainly because I don't have any idea what too use the remaining keys for. However the keyboard has 46 keys since I did not remove those 3 unused keys. So 46 makes more sense to me. If there was a second parameter "usedKeyCount", it should be 43.